### ISSUE TITLE
Fix unchecked types warnings.

### ISSUE THREAD

**Issue by cbosdo** [uyuni-project/uyuni#612]
There is a lot of java code that doesn't use the generic types... and this generates thousands of warnings. For sure this task is a one that will take time, but can be easily split.

**Comment by Shirocco88** [comment]
@cbosdo :  I think this issue can be closed,

**Comment by cbosdo** [comment]
No, there are still a lot of those and they are easy to fix for someone new to the code. This issue is kind of a long running one that would be fixed into smaller PRs.

**Comment by cbosdo** [comment]
The list of the remaining occurrences of this error can be found on SonarCloud:
https://sonarcloud.io/project/issues?languages=java&resolved=false&rules=java%3AS3740&id=uyuni-project_uyuni

**Comment by Bhavya031** [comment]
@cbosdo I don’t think this is closed. There are still 707 warnings left, and I might open some smaller PRs to update the code
